jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/internal/ContentCopier.java

```diff
@@ -51,9 +51,6 @@ protected Action process() throws Throwable
             return Action.IDLE;
         }
 
-        if (current instanceof Content.Chunk.Error error)
-            throw error.getCause();
```

Comment:
Is Content.Chunk.Error handled somewhere else now?

Reply:
Yes. Implementations of `Sink.write(Chunk, Callback)` will fail the callback if they receive a `Chunk.Error`.